roles/edpm_ovn/defaults/main.yml

```diff
@@ -31,7 +31,7 @@ edpm_ovn_chassis_mac_mapping_prefixes:
 edpm_ovn_chassis_mac_mapping_seed: "{{ ansible_machine_id }}"
 edpm_ovn_encap_type: geneve
 edpm_ovn_dbs: []
-edpm_enable_dvr: true
+edpm_enable_chassis_gw: false
```

Comment:
Everything seems good to me. But I'm wondering about this flip. Is the fact that the chassis gw is disabled by default, rather than previous situation when it was enabled, properly documented?

Reply:
I thought the intention here was to have DVR enabled by default and not chassis-gws. So I enabled by default DVR in https://github.com/openstack-k8s-operators/neutron-operator/pull/153, and make this only about the chassis gw -- which I think the intention was to only have it by default on the controllers. 
It was in fact highlighed by Yatin that unless that option is set on the controllers (enable-chassis-as-gw), all the nodes (computes and controllers) will be used anyway for this.

Reply:
>I thought the intention here was to have DVR enabled by default and not chassis-gws. So I enabled by default DVR in https://github.com/openstack-k8s-operators/neutron-operator/pull/153, and make this only about the chassis gw 
In that case it's all fine. Maybe it would be better to add that info commit message next time.